### PR TITLE
DFTDP # 211 Partner Portal Dashboard

### PR DESCRIPTION
**Pull request by shruthi-rapala**
- Added search and remedial componneys
- Added routes in the nav componnet
- Added driver lookup section
- Added remedial section
- Added Need assistance section on dashboard

# Description

This PR includes the following proposed change(s):

- {List all the changes, if possible add the jira ticket #}

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

### Screenshots

Include screen captures or recordings with captions of the tests run (unit tests, postman request/response and or manual UI driven tests.

## Does the change impact or break the Docker build?

- [ ] Yes
- [ ] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [ ] My code follows the [style guidelines](https://justice.gov.bc.ca/wiki/display/JAM/RCC+Style+Guide) of this project
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: partner-portal/src/UI/src/app/app.routes.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { Routes } from '@angular/router';
+import { RemedialComponent } from '@app/remedial/remedial.component';
+import { SearchComponent } from '@app/search/search.component';
+
+export const ROUTE_DENIED = 'denied';
```
Comment (jburditt) [inline partner-portal/src/UI/src/app/app.routes.ts:5]: does partner portal have a denied page?

File: partner-portal/src/UI/src/app/search/search.component.scss [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    flex-direction: row; 
+    gap: 32px;
+    padding-top: 32px;
+  }
+  .search-content{
+    display: flex;
+    flex-direction: row;
+  }
+
+  .driver{
```
Comment (jburditt) [inline partner-portal/src/UI/src/app/search/search.component.scss:40]: I'm worried we are gonna have leftover CSS from temporary code

**Review by jburditt — APPROVED** [review]

File: partner-portal/src/UI/src/app/app.routes.ts (hunk 1 above)
Comment (shruthi-rapala) [reply partner-portal/src/UI/src/app/app.routes.ts:5]: For now we don't have the denied Route but might need that. For now i will remove that and add that when it is required.

File: partner-portal/src/UI/src/app/search/search.component.scss (hunk 2 above)
Comment (shruthi-rapala) [reply partner-portal/src/UI/src/app/search/search.component.scss:40]: This is a new page and these are the new styles added for this page not related to any previous pages on other portals.

**Pull request merged**